apps/client/src/pages/Dashboard/chartAndUpcoming/Chart/CustomTootip.tsx

```diff
@@ -0,0 +1,46 @@
+import { TooltipProps } from 'recharts'
+import PropTypes, { InferProps } from 'prop-types'
+import styles from './chat.module.scss'
+import Indicator from './images/Legend Indicator.svg?react'
+import Beak from './images/Bottom Beak.svg?react'
+import { Text } from 'components'
+
+interface DataPoint {
+  value?: number
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
was there no way to type this?

Reply:
There should definitely be a way, but hasn’t figure it out yet